eve/main.yml

```diff
@@ -442,13 +446,15 @@ models:
         TEST_HOSTS_LIST: "bootstrap"
         PYTEST_FILTERS: "post and ci"
         BOOTSTRAP_BACKUP_ARCHIVE: ""
+        CONTROL_PLANE_INGRESS_VIP: "192.168.1.253"
```

Comment:
Shouldn't it be:

```suggestion
        CONTROL_PLANE_INGRESS_VIP: "192.168.1.254"
```

, as defined in BootstrapConfig above?

Reply:
Nop, it's the goal here, it's to change the VIP to something else

Reply:
I see, OK.